### PR TITLE
Added comment notifier to youtrack worflows

### PR DESCRIPTION
**Pull request by christopherbauer**
@kradima Would you guys be able to use this, or is this repository closed to pull requests?

### CODE REVIEW

File: Comment Notifier/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+```java
+rule Notify All Mentioned Users In Comments 
+ 
+when comments.added.last.text.contains("@", ignoreCase) { 
+  var mentionedUsers = comments.last.text.split("@", opts); 
+  for each mentionedUser in mentionedUsers { 
+    var mentionedUserEndIndex = mentionedUser.indexOf(" ", opts); 
+    var mentionedUserName = mentionedUser.substring(0, mentionedUserEndIndex); 
+    for each user in {group: All Users}.getUsers() { 
+      if (user.login == mentionedUserName) {
```
Comment (kradima) [inline Comment Notifier/README.md]: It's better to compare strings by 'eq()' method.

**Comment by kradima** [comment]
I think multiple users notifying could be usefull, thank you.

**Comment by christopherbauer** [comment]
I committed an update to use .eq instead of the equality operator.

**Comment by christopherbauer** [comment]
Ooh, sorry. Fixed!

**Pull request merged**